osquery/tables/applications/windows/office_most_recently_used.cpp

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <osquery/core/core.h>
+#include <osquery/core/tables.h>
+#include <osquery/filesystem/fileops.h>
+#include <osquery/logger/logger.h>
+#include <osquery/tables/system/windows/registry.h>
+#include <osquery/utils/conversions/tryto.h>
+#include <osquery/utils/conversions/windows/windows_time.h>
+
+#include <string>
+
+namespace osquery {
+namespace tables {
+
+// Get all Office applications and get all installed versions
+constexpr auto kOfficePath = "\\Software\\Microsoft\\Office\\%\\%\\File MRU\\%";
+constexpr auto kOffice365Path =
+    "\\Software\\Microsoft\\Office\\%\\%\\User MRU\\%";
```

Comment:
The implication here, is that this is a registry entry, and that this table primarily reformats the registry for nicer display?

Is that correct? If so, I'm ambivalent... One of the things I've found, is that it can lead to a large number of specific tables, where I favor a smaller number of compostable tables. Another issue is that it can create maintenance burdens. This cites 2 different registry paths. What happens if there's a third?

Reply:
I'm withdrawing this objection. We talked about in office hours, and there is not consensus. Table seems useful!